wls-wahlvorbereitung-service/src/main/resources/db/migrations/h2/V4_0__createTableEroeffnungsuhrzeit.sql

```diff
@@ -0,0 +1,6 @@
+create table Eroeffnungsuhrzeit
+(
+    wahlbezirkid          varchar(1000) not null,
+    eroeffnungsuhrzeit TIMESTAMP     not null,
+    primary key (wahlbezirkid)
```

Comment:
Hat IntelliJ bei dir nicht formartiert?
```suggestion
    wahlbezirkid       varchar(1000) not null,
    eroeffnungsuhrzeit TIMESTAMP     not null,
    primary key (wahlbezirkid)
```

Reply:
ich habe spotless apply laufen lassen, aber nicht gemacht.